src/v2/stack_aggregator.py

```diff
@@ -401,8 +401,13 @@ def initiate_unknown_package_ingestion(aggregator: Aggregator):
     ecosystem = aggregator._normalized_packages.ecosystem
     try:
         for dep in aggregator.get_all_unknown_packages():
-            server_create_analysis(ecosystem, dep.name, dep.version, api_flow=True,
-                                   force=False, force_graph_sync=True)
+            if Settings().disable_unknown_package_flow:
```

Comment:
Why it is been changed to check `disable_unknown_package_flow` for each package? Actually the earlier approach of checking the condition at the beginning of the function was good, except that you can use early return.
```
            if Settings().disable_unknown_package_flow:
                logger.warning('Skipping unknown flow %s', aggregator.get_all_unknown_packages())
                return
```

Reply:
I did this to get the details of package name and version that were skipped, as asked by previous comment.

Reply:
I dont think we need to log every pkg or version we miss.. if needed we can either print the msg that its disabled or print the entire list. printing a log for every iteration looks useless to me